tests/Aspire.Hosting.Analyzers.Tests/Aspire.Hosting.Analyzers.Tests.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+
+    <RunTestsOnHelix>false</RunTestsOnHelix>
```

Comment:
Is there a reason these can't be run on Helix? I think our goal is to get them all running there, if we can.

cc @radical

Reply:
Hmmm I'm not sure where that came from actually. I can try removing.